### DISCUSSION TITLE
请求数据访问

### DISCUSSION THREAD

**Discussion by lightsing** [NathanKun/resonance-columba#139] [General]
@NathanKun 您好，我想基于价格数据做一个衍生的小游戏，可以请求一个 Firebase 的只读权限吗？

**Comment by NathanKun** [comment]
你好，
抱歉，不太方便，因为firebase是按读取计费的。
不过你可以自行调取网站的api：
https://www.resonance-columba.com/api/get-prices
注意调用频率不要太高就好。
并使用 utils/price-api-compressor.ts 里的 revertLowBandwidthData 函数将数据还原为可读形态。

**Reply by lightsing** [reply]
好的感谢

**Reply by lightsing** [reply]
另外，现在的数据都是手动更新的吗？

**Reply by NathanKun** [reply]
主要是自动，但也可手动。

**Reply by winerchan** [reply]
您好，因为盯商品价格比较花时间，想做一个简单的程序根据价格进行判断并向我推送消息，可以使用网站已有的获取价格api嘛？
仅个人使用不准备公开，频率大概会控制在3-5min一次（如果需要的话可以使用特殊请求头用于区分）

**Reply by NathanKun** [reply]
@winerchan  你好，可以的。

**Reply by winerchan** [reply]
> @winerchan 你好，可以的。

好的，谢谢
